main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Suggest updating while loop to a for loop since range is known.

<img width="436" alt="image" src="https://user-images.githubusercontent.com/72112832/201221050-04c5edce-b6eb-463b-b75a-f8eeb5d0a561.png">